integration/command.go

```diff
@@ -56,6 +56,9 @@ type command struct {
 	// expectedRegexp contains a regex used to match against the command output.
 	expectedRegexp string
 
+	// customOutputVerifier is a function that defines a custom verifier.
+	customOutputVerifier func(output string) error
```

Comment:
I'm not really good a naming things, but what about the following options?

`expectedOutput`
`expectedOutputFn`
`expectedOutputVerifier`
`expectedOutputChecker`

?

Reply:
Yeah, they follow the slipstream from `expectedRegexp` and `expectedString`. I will go with `expectedOutputFn`.